test/accessibility/.gitignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
Pa11y no longer outputs html reports, which is why this was removed. Guessing you had some old ones in your local source? So worth leaving this in?